### DISCUSSION TITLE
Material.coordinate_system

### DISCUSSION THREAD

**Discussion by rickygill00** [ansys/pyaedt#4782] [Q&A] [answered]
Hello,

For the Material class is it possible to change the three components of the coordinate system? For example, I can change from "Cartesian" to "Cylindrical" by doing Material.coordinate_system = "Cylindrical". But if I want to change the "R Component" from 1 to -1 how would I do it?

**Comment by gmalinve** [comment, marked as answer]
Hi @rickygill00,

Thanks for opening this discussion.
You can get access to the magnetic coercivity of the material and then update the cylindrical coordinate system as:
`m2d.materials.material_keys[mat_name].magnetic_coercivity.value = ["0", "-1", "0", "0"]`
and for safety use the update(): 
`m2d.materials.material_keys[mat_name].update()`

Hope this helps,
Regards,

Giulia
